mappings/net/minecraft/util/thread/MessageListener.mapping

```diff
@@ -7,3 +7,5 @@ CLASS net/minecraft/class_3906 net/minecraft/util/thread/MessageListener
 		ARG 1 action
 	METHOD method_17345 ask (Ljava/util/function/Function;)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 messageProvider
+	METHOD method_27918 askFallible (Ljava/util/function/Function;)Ljava/util/concurrent/CompletableFuture;
```

Comment:
askFallible as in?

Reply:
seems this is near-identical to `ask` but it can return a `CompletableFuture` that completes exceptionally, wheras `ask` will never(?) do that

Reply:
It would be nice if you could make that distinguishment in the javadoc

Reply:
Added in e9f0cea